src/main/g8/$name__norm$-stream-impl/src/main/resources/application.conf

```diff
@@ -27,8 +27,3 @@ lagom.persistence.read-side.cassandra.keyspace = \${$name;format="norm"$-stream.
 # Prefer 'ddata' over 'persistence' to share cluster sharding state for new projects.
 # See https://doc.akka.io/docs/akka/current/cluster-sharding.html#distributed-data-vs-persistence-mode
 akka.cluster.sharding.state-store-mode = ddata
-
-# Enable the serializer for akka.Done provided in Akka 2.5.8+ to avoid the use of Java serialization.
-akka.actor.serialization-bindings {
-  "akka.Done" = akka-misc
```

Comment:
This is only available if `lagomJavadslCluster` is on the project deps. As it was this was causing an Exception on startup time that prevents the `hello-stream-impl` from starting.

The issue was hidden by https://github.com/lagom/lagom/issues/1222